src/components/app.js

```diff
@@ -303,23 +329,56 @@ const App = () => {
                         user={user}
                         appsAndUrls={appsAndUrls}
                     >
-                        <Tabs
-                            value={selectedTabId}
-                            indicatorColor="primary"
-                            variant="scrollable"
-                            scrollButtons="auto"
-                            onChange={(event, newValue) => toggleTab(newValue)}
-                            aria-label="parameters"
-                            className={classes.process}
-                        >
-                            {configs.map((config) => (
-                                <Tab
-                                    key={config.process}
-                                    label={config.process}
-                                    value={config.process}
-                                />
-                            ))}
-                        </Tabs>
+                        <div className={classes.child}>
+                            <Tabs
+                                value={
+                                    configs
```

Comment:
why check again here that selectedTabId is in the configs ? It's already done in the useEffect above ?

Reply:
after deleting the current process and before selectedTabId gets updated, we should pass false to tabs value, else we get the following error:
"The value provided to the Tabs component is invalid.
None of the Tabs' children match with `OLD_PROCESS_NAME`."